src/components/ConferencePanel/index.js

```diff
@@ -0,0 +1,198 @@
+import React, { PropTypes, Component } from 'react';
+import formatMessage from 'format-message';
+import formatNumber from 'ringcentral-integration/lib/formatNumber';
+import IconField from '../IconField';
+import Switch from '../Switch';
+import i18n from './i18n';
+import styles from './styles.scss';
+import RcFont from '../../assets/RcFont/RcFont.scss';
+
+class ConferencePanel extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showInternational: false,
+      searchInternationals: this.props.conferenceNumbers.phoneNumbers,
+      selectInternationals: [],
+    };
+    this.formatInternational = (phoneNumber, countryCode) => {
+      if (phoneNumber.indexOf(countryCode === 1)) {
+        return `+${countryCode} ${phoneNumber.replace('+', '').replace(countryCode, '')}`;
+      }
+      return phoneNumber;
+    };
+    this.formatPin = (number) => {
+      if (!number) {
+        return '';
+      }
+      return number.replace(/(\d{3})/g, '$1-').replace(/-$/, '');
+    };
+
+    this.formatNumbers = {
```

Comment:
But I think the formatNumbers function is only to display the formatted number,..and same numbers have different types to display..so I put in the presentational component.